pyproj/_crs.pxd

```diff
@@ -22,7 +22,14 @@ cdef class AreaOfUse:
     @staticmethod
     cdef create(PJ_CONTEXT* projcontext, PJ* projobj)
 
-cdef class Ellipsoid:
+
+cdef class BaseCRS:
```

Comment:
Small nitpick: I would not call this `BaseCRS`, as it is not the base of only the CRS classes

Reply:
I don't disagree, what were you thinking for a name?

Reply:
In the C++ code this is called `IdentifiedObject`, if we would like to get ispiration from there, but for me something like `Base` or `Common` is also fine